### PR TITLE
[pr2eus] Add end-coords-interpolation

### PR DESCRIPTION
**Pull request by pazeshun**
Fixed version of #244

### CODE REVIEW

File: pr2eus/robot-interface.l [hunk 1]
```diff
@@ -356,7 +356,10 @@
 - start-time : time to start moving
 - scale : if tm is not specified, it will use 1/scale of the fastest speed
 - min-time : minimum time to for time to goal
+- end-coords-interpolation : set t if you want to move robot in cartesian space interpolation
 "
+   (if end-coords-interpolation
+     (return-from :anlge-vector (send self :angle-vector-sequence (list av) (list tm) ctype start-time :scale scale :min-time min-time :end-coords-interpolation t)))
```
Comment (knorth55) [inline pr2eus/robot-interface.l]: typo `anlge-vector`

**Comment by pazeshun** [comment]
Fixed

File: pr2eus/robot-interface.l [hunk 2]
```diff
@@ -455,6 +455,10 @@
                    (dolist (limb target-limbs)
                      (setq ec-prev (elt end-coords-prev (position limb limbs))
                            ec-current (elt end-coords-current (position limb limbs)))
+                     ;; set midpoint of av as initial pose of IK
```
Comment (k-okada) [inline pr2eus/robot-interface.l]: I think you have to set this before `(dolist (limb target-limb)`, Please check with dual arm case.
https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/273/commits/19019cba7aaaf4a26a5f3efdc396610b1bf0c887#diff-15f6920b924513f6027d36b361f85bbdR455

**Comment by pazeshun** [comment]
Fixed

**Pull request merged**